xchange-stream-service-netty/src/main/java/info/bitrich/xchangestream/service/netty/NettyStreamingService.java

```diff
@@ -419,6 +447,45 @@ public void sendMessage(String message) {
         .share();
   }
 
+  public Observable<T> subscribeChannelMultipleMessages(String channelName, Object... args) {
```

Comment:
rather then add more methods to this service class can you just use existing methods? I believe binance does this?